academy/ibc-dev/ibcapp_packet.md

```diff
@@ -349,33 +354,43 @@ const (
 	AttributeKeyAckError   = "error"
 )
 ```
-Here we find both the `Event` type and the attributes it contains.
+Here are found both the `Event` type and the attributes it contains.
 
-These are not the only relevant events for IBC though, the others can be found in the core IBC source code:
+These are not the only relevant events for IBC, though, the others can be found in the core IBC source code:
 - [client events](https://github.com/cosmos/ibc-go/blob/main/modules/core/02-client/types/events.go)
 - [connection events](https://github.com/cosmos/ibc-go/blob/main/modules/core/03-connection/types/events.go)
 - [channel events](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/types/events.go)
 
-You can go back to the code we've examined so far to take note of the events emitted.
+You can go back to the code examined so far to take note of the events emitted.
 
 </HighlightBox>
 
-## Summary
+<HighlightBox type="note">
 
-Let's summarize what we've accomplished so far:
-- we've scaffolded a chain with an IBC enabled module, both chain and module called leaderboard
-- Ignite CLI made sure to implement the `IBCModule` interface including channel handshake and packet callbacks
-- Ignite CLI has bound our IBC module to a port and added a route to the IBC router
-- we've scaffolded an IBC packet, `IbcTopRankPacket`
-- Ignite CLI defined the packet and ack data
-- Ignite CLI has set up the basic message handling and packet handling to send, receive, acknowledge and timeout packets
```

Comment:
I'd argue to keep the summary here. The text below directly follows from a consideration of what was already achieved and what is still to be done/out of scope + links to further steps